tests/scripts/test-e2e-standalone.sh

```diff
@@ -51,8 +55,15 @@ if [ ! -e "$PROVIDER_KEY_PATH" ]; then
 fi
 
 export MODEL="${MODEL:-gpt-3.5-turbo-1106}"
+if [ -z ${TEST_HTTP_HEADER_REDACTION:-} ]; then
+  LOG_LEVEL=info
+else
+  LOG_LEVEL=debug
+fi
+export LOG_LEVEL
```

Comment:
we can just run OLS w/ debug level logging in all our e2e jobs (cluster and standalone). I thought we already were.